extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -180,7 +180,7 @@ public static enum Method {
             }
 
             /**
-             * The client secret value - it will be ignored if 'secret.key' is set
+             * The client secret value - it will be ignored if 'quarkus.oidc.credentials.secret' is set
```

Comment:
@michalvavrik The problem here is that it wont work with `quarkus-oidc-client`, so somehow we should mention both options

Reply:
okay, I didn't realize that. But if I couldn't figure this after looking into code for several minutes, I'm not sure how could users be expect to understand that. I'll think about it...

Reply:
What about `credentials.secret`? it's not perfect, but at least it exists.

I pushed that change, should you have better idea, please let me know.

Reply:
@michalvavrik sounds good